tools/swift-reflection-fuzzer/swift-reflection-fuzzer.cpp

```diff
@@ -28,6 +28,7 @@
 #include "llvm/Object/ELFObjectFile.h"
 #include "llvm/Object/MachOUniversal.h"
 #include "llvm/Support/CommandLine.h"
+#include <stdio.h>
```

Comment:
This is a cpp file. Can you please use `cstdio` and also change stddef/stdint to also use the proper c++ include.

Reply:
Funnily enough, I started using `cstdio` and then assumed that that wasn't the right thing because of the other headers. So yes, let's do that instead.